gating/pre_merge_test/run_deploy_mnaio.sh

```diff
@@ -111,14 +111,20 @@ env | grep RE_ | sed 's/^/export /' > /opt/rpc-openstack/RE_ENV
 
 # check if we're using artifacts or not
 if [[ ${RE_JOB_IMAGE} =~ no_artifacts$ ]]; then
-  export RPC_APT_ARTIFACT_ENABLED=no
+  export RPC_APT_ARTIFACT_ENABLED="no"
   echo "export RPC_APT_ARTIFACT_ENABLED=no" >> /opt/rpc-openstack/RE_ENV
   ${MNAIO_SSH} "apt-get -qq update; DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade"
+
 elif [[ ${RE_JOB_IMAGE} =~ loose_artifacts$ ]]; then
   # Set the apt artifact mode
-  export RPC_APT_ARTIFACT_MODE=loose
+  export RPC_APT_ARTIFACT_MODE="loose"
   echo "export RPC_APT_ARTIFACT_MODE=loose" >> /opt/rpc-openstack/RE_ENV
   ${MNAIO_SSH} "apt-get -qq update; DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade"
+
+elif [[ ${RE_JOB_IMAGE} =~ strict_artifacts$ ]]; then
+  # Set the apt artifact mode
+  export RPC_APT_ARTIFACT_MODE="strict" >> /opt/rpc-openstack/RE_ENV
+  export RPC_APT_ARTIFACT_ENABLED="yes" >> /opt/rpc-openstack/RE_ENV
```

Comment:
I don't think this command produces output. Did you mean to use the same command sequence as above?